docs: simplify component descriptions (#294)
arcus-azure/arcus: website/resources/components/azure-ml.md

---
title: Arcus Azure ML

> description: Streamline Azure ML development, but lets ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.

Reviewer: ```suggestion
description: Streamline Azure ML development and lets ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.
```

maybe this is a little better?

Reviewer 2: ```suggestion
description: Streamline Azure ML development and let ML engineers focus on the actual job at hand, without losing time in tinkering with the AzureML SDK.
```
What about this?
